Client/Frontend/Menu/AppMenuConfiguration.swift

```diff
@@ -166,6 +179,14 @@ extension AppMenuConfiguration {
         return AppMenuItem(title: ViewMobileSiteTitleString, action:  MenuAction(action: AppMenuAction.ToggleBrowsingMode.rawValue), icon: "menu-ViewMobile", privateModeIcon: "menu-ViewMobile-pbm")
     }
 
+    private static var NoImageModeItem: MenuItem {
+        return AppMenuItem(title: "无图模式", action:  MenuAction(action: AppMenuAction.ToggleNoImageMode.rawValue), icon: "menu-NoImageMode", privateModeIcon: "menu-NoImageMode-pbm")
+    }
```

Comment:
`NSLocalizedString` for all user facing strings. Even if you don't think this well ever be localized.